Makefile

```diff
@@ -16,12 +16,23 @@ all: hubble
 hubble:
 	$(GO) build $(if $(GO_TAGS),-tags $(GO_TAGS)) -ldflags "-w -s -X 'github.com/cilium/hubble/pkg.GitBranch=${GIT_BRANCH}' -X 'github.com/cilium/hubble/pkg.GitHash=$(GIT_HASH)' -X 'github.com/cilium/hubble/pkg.Version=${VERSION}'" -o $(TARGET)
 
+release:
+	for OS in darwin linux; do \
+		for ARCH in 386 amd64; do \
+			test -d release/$$OS/$$ARCH|| mkdir -p release/$$OS/$$ARCH; \
+			env GOOS=$$OS GOARCH=$$ARCH $(GO) build $(if $(GO_TAGS),-tags $(GO_TAGS)) -ldflags "-w -s -X 'github.com/cilium/hubble/pkg.Version=${VERSION}'" -o release/$$OS/$$ARCH/$(TARGET); \
```

Comment:
nit: Can we extract a common variable/macro for the build flags (-tags, -w, -s, -X etc)?